src/Core/Managed/Shared/TelemetryClientExtensions.cs

```diff
@@ -32,8 +35,9 @@ public static IOperationHolder<T> StartOperation<T>(this TelemetryClient telemet
         /// <param name="operationName">Name of the operation that customer is planning to propagate.</param>
         /// <param name="operationId">Operation ID to set in the new operation.</param>
         /// <param name="parentOperationId">Optional parent operation ID to set in the new operation.</param>
+        /// <param name="correlationContext">CorrelationContext that is added to telemetry properties and propagated with outgoing HTTP calls.</param>
```

Comment:
I was thinking about it and want to remove this parameter at all: there is no way get correlation-context (as TelemetryContext.CorrelationContext is internal) and should be no way to  set it in user code.
If there is a Correlation-context from upstream service, we just propagate it further. If  AI SDK needs to pass something for internal needs - it's always there as internal.